src/olympia/lib/settings_base.py

```diff
@@ -1060,21 +1060,14 @@ def get_db_config(environ_var, atomic_requests=True):
 # a separate, shorter timeout for validation tasks.
 CELERY_TASK_SOFT_TIME_LIMIT = 60 * 30
 
-# List of modules that contain tasks and that wouldn't necessarily be imported
-# automatically when celery starts.
+# List of modules that contain tasks and that wouldn't be autodiscovered by
+# celery. Typically, it's either `tasks` modules from something not in
+# INSTALLED_APPS, or modules not called `tasks`.
 CELERY_IMPORTS = (
-    'olympia.activity.tasks',
-    'olympia.bandwagon.tasks',
-    'olympia.files.tasks',
     'olympia.lib.crypto.tasks',
     'olympia.lib.es.management.commands.reindex',
-    'olympia.ratings.tasks',
-    'olympia.reviewers.tasks',
-    'olympia.versions.tasks',
     'olympia.stats.management.commands.index_stats',
-    'olympia.tags.tasks',
-    'olympia.users.tasks',
-    'olympia.zadmin.tasks',
+    'olympia.lib.es.tests.test_commands',
```

Comment:
mmm, could that line be appended to the list in `settings_test` to avoid specifying test code in `settings_base`?

Reply:
Yeah, didn't think about that but that's a good point.